app/models/service_template_ansible_tower.rb

```diff
@@ -6,6 +6,18 @@ class ServiceTemplateAnsibleTower < ServiceTemplate
   alias_method :job_template, :configuration_script
   alias_method :job_template=, :configuration_script=
 
+  def self.create_catalog_item(options, _auth_user = nil)
+    transaction do
+      create(options.except(:config_info)) do |service_template|
+        config_info = validate_config_info(options)
+
+        service_template.job_template = ConfigurationScript.find(config_info[:configuration_script_id])
```

Comment:
We want to accept `:configuration_script_id` or `:configuration`. The later points to an object that can be directly assigned. Same to manager. Need to update the validate method too.